core/templates/components/common-layout-directives/common-elements/confirm-or-cancel-modal.controller.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for simple modal with only two actions: close or
+ * dismiss.
+ */
+
+angular.module('oppia').controller('ConfirmOrCancelModalController', [
+  '$scope', '$uibModalInstance',
+  function($scope, $uibModalInstance) {
+    $scope.confirm = function(message) {
```

Comment:
Maybe `value`? (message seems like a string value but this can be anything, right?)

Reply:
Nice catch! Done.